### ISSUE TITLE
Readme is inconsistent with implementation

### ISSUE THREAD

**Issue by Miguel-Serejo** [WildSideUK/LaravelMailgunEu#1]
The readme instructs users to provide a full url in the `domain` setting, but the actual implementation is hard-coding the EU endoint prefix to the domain value.

If the intent of this package is simply to allow drop-in replacement of the API endpoint, I suggest rewording the readme to be clear that this is all it does and doesn't offer any further configuration options, nor does it require the config values to be changed.

**Comment by mattmcdev** [comment]
I'd committed the wrong version of the file, should be fixed now.
